Robus/src/msg_alloc.c

```diff
@@ -562,8 +562,7 @@ static inline error_return_t MsgAlloc_ClearMsgSpace(void *from, void *to)
  ******************************************************************************/
 static inline error_return_t MsgAlloc_CheckMsgSpace(void *from, void *to)
 {
-    if ((((uint32_t)used_msg >= (uint32_t)from) && ((uint32_t)used_msg <= (uint32_t)to)) ||
-        (((uint32_t)oldest_msg >= (uint32_t)from) && ((uint32_t)oldest_msg <= (uint32_t)to)))
+    if ((((uint32_t)used_msg >= (uint32_t)from) && ((uint32_t)used_msg <= (uint32_t)to)) || (((uint32_t)oldest_msg >= (uint32_t)from) && ((uint32_t)oldest_msg <= (uint32_t)to)))
```

Comment:
Perhaps we should put OR operator || in the second line.  So, after clang format processing, the conditions should be on different lines.